repository.eselect.5.in

```diff
@@ -0,0 +1,169 @@
+.\" -*- coding: utf-8 -*-
+.\" Copyright 2019 Till Schäfer
+.\" Distributed under the terms of the 2-clause BSD license
+.\"
+.TH repository.eselect 5 "31 January 2019" "Gentoo Linux" "eselect"
+.SH NAME
+\fBrepository.eselect\fP - Manage repos.conf via eselect
+.SH SYNOPSIS
+.nf
+.fam C
+\fBeselect\fP \fBrepository\fP [\fBhelp\fP|\fBusage\fP|\fBversion\fP]
+\fBeselect\fP \fBrepository\fP \fBadd\fP \fIname\fP \fIsync-type\fP \fIsync-uri\fP
```

Comment:
I think it's customary to put user parameters in `<...>`.

Reply:
AFAIK this is not true for man pages (they are just underlined there / gross: \fI). At least this is the was the man page of portage is written and the man pages of man itself. Thus, I think it is better to keep in line with the other documentations of related tools.